README.md

```diff
@@ -111,6 +112,22 @@ Alternatively you can use
 
 NB Note that `pip` will not set up PyTorch with CUDA and GPU support. Therefore, we recommend to use `conda`.
 
+### Test 
+After cloning the [current](https://github.com/esa/torchquad.git) repository, developers can test the correct installation of `torchquad` by running 
```

Comment:
What do you mean by current?

Would just write
After cloning the repository

Also, I think you need to run pytest in torchquad/tests or the imports will not work?